modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices_{context}"]
+= Adding kernel arguments to enable IOMMU for the BIOS
+
+To enable an IOMMU extension for the BIOS (Basic Input/Output System) associated with a PCI device, create the `MachineConfig` object file and add the kernel arguments for the device.
+
+.Prerequisites
+. Administrative privilege to a working {product-title} cluster.
+. The CPU for the device hardware belongs to either Intel or AMD brand.
+. Enable the hardware virtualization extensions for Intel Virtualization Technology (Intel VT-X). AMD-V extensions are typically enabled by default.
+
+[NOTE]
+====
+If you have an AMD CPU, specify the kernel argument as `amd_iommu=on`.
+====
+
+.Procedure
+. Create a `MachineConfig` object file that identifies the kernel argument. The following example shows a kernel argument for an Intel CPU.
+
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig <1>
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: worker <2>
+    name: 100-worker-iommu <3>
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+    kernelArguments:
+        - intel_iommu=on <4>
+...
+----
+<1> The `MachineConfig` for the kernel argument.
+<2> Applies the new kernel argument only to worker nodes.
+<3> The `name` indicates the ranking of this kernel argument (100) among the machine configs and its purpose.
+<4> Identifies the kernel argument as `intel_iommu` for an Intel CPU.
+
+. Create the new `MachineConfig` object file:
++
+[source,terminal]
+----
+$ oc create -f 100-worker-kernel-arg-iommu.yaml
+----
+
+.Verification
+
+. Verify that the new `MachineConfig` object was added.
```

Comment:
For a single step, our style is to use a bullet. (Finding this specific reference in our guidelines is failing me at the moment, sorry!)

Reply:
Updated the content - changed single unordered item into a bullet.